tests/wasip2.json

```diff
@@ -0,0 +1,4 @@
+{
+	"inherits": ["wasip2"],
+	"emulator": "wasmtime --wasm component-model -Sinherit-network -Sallow-ip-name-lookup --dir={tmpDir}::/tmp {}"
```

Comment:
For runtime tests that don't require networking, can we customize this JSON to suit specific test cases?

Reply:
I submitted a PR to TinyGo to make this unnecessary.

Reply:
https://github.com/tinygo-org/tinygo/pull/4468